Kitodo-FileManagement/pom.xml

```diff
@@ -44,12 +44,23 @@
             <artifactId>kitodo-api</artifactId>
             <version>3.0.0-beta.1-SNAPSHOT</version>
         </dependency>
+        <dependency>
+            <groupId>org.apache.commons</groupId>
+            <artifactId>commons-lang3</artifactId>
+            <version>${commons-lang3.version}</version>
+        </dependency>
         <dependency>
             <groupId>junit</groupId>
             <artifactId>junit</artifactId>
             <version>${junit.version}</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
```

Comment:
please use junit, we do not need several testing libraries. Or which aspect do you need, junit doesn't offer?

Reply:
There is a long explanation from me in the corresponding ticket in the Jira, including examples. Abstract: It provides much better error descriptions when a test fails. Many tests can be written shorter and more readable.
Either way, that's a separate topic. We should discuss it there and decide how we handle it.